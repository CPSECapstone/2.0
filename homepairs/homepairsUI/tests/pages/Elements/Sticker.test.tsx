/**
 * @jest-environment jsdom
 */

import {Sticker} from 'homepairs-elements';
import { shallow } from 'enzyme';
import * as React from 'react';
import { View, Text} from 'react-native';


describe("Sticker", () => {
  
    const rendered = shallow(<Sticker><Text>Hello World!</Text></Sticker>);
    const rendered2 = shallow(<Sticker><Text>Hello World!</Text><View/><View/></Sticker>);

    it("Method Test", () => {
      expect(rendered.find(Text)).toHaveLength(1);
      expect(rendered2.find(View)).toHaveLength(3);
      expect(rendered2.find(Text)).toHaveLength(1);
      
      // Test to make sure no unexpected changes occured. This is an element so this should always pass 
      expect(rendered2).toMatchSnapshot();
    });
    
  });