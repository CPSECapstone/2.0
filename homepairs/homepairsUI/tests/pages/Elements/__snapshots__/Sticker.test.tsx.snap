// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sticker Method Test 1`] = `ShallowWrapper {}`;
